bee-common/bee-common-derive/tests/packable/fail/missing_tag_enum.stderr

```diff
@@ -0,0 +1,5 @@
+error: All variants of an enum that derives `Packable` require a `#[packable(tag = ...)]` attribute.
```

Comment:
```suggestion
error: All variants of an enum that derive `Packable` require a `#[packable(tag = ...)]` attribute.
```

Reply:
but the enum is the one deriving packable. Maybe my grammar is wrong?